### PR TITLE
Input validation VR488 

### PR DESCRIPTION
**Pull request by Mech0z**
Added new VatClassification.Unknown as default value, added VR488 VatClassificationValidationRule.cs and unittests of that rule

## Description
Input Validation of VR488

- Only Vat classification allowed is D01 Vat or D02 NoVat

Unknown value added to VatClassification to avoid default values ending up as valid input

## References

* #195

### CODE REVIEW

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/VatClassificationValidationRuleTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    [UnitTest]
+    public class VatClassificationValidationRuleTests
+    {
+        [Theory]
+        [InlineAutoMoqData("NoVat", true)]
+        [InlineAutoMoqData("Vat25", true)]
+        [InlineAutoMoqData("Unknown", false)]
+        public void VatClassificationValidationRule_Test(
+            string vatClassificationAsString,
```
Comment (lasrinnil) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/VatClassificationValidationRuleTests.cs:32]: I think we should use the VatClassification enum as parameter for this test. This way we will also end up using the unknown value, and we can remove the resharper comment.

Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/VatClassificationValidationRuleTests.cs:32]: Changed to using enum

**Review by bemwen — APPROVED** [review]

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Validation/ValidationRuleIdentifier.cs [hunk 2]
```diff
@@ -23,5 +23,6 @@ public enum ValidationRuleIdentifier
         SenderIsMandatory, // VR150
         RecipientIsMandatory, // VR153
         ChargeOperationIdIsRequired, // VR223
+        VatClassification, // VR488
```
Comment (lasrinnil) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Validation/ValidationRuleIdentifier.cs:26]: This is not explicit enough. It is basically the name of a Enum we use.

Comment (Mech0z) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Validation/ValidationRuleIdentifier.cs]: Changed to InvalidVatClassificationOnCreate

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**